docs/website/blog/2022-12-13-odo-v3.4.0.md

```diff
@@ -50,7 +50,6 @@ Example:
 #### Using `odo` with Podman
 
 Based on your valuable feedback, we have worked on improving the experience of using `odo` with Podman.
-Note that this is still [experimental](../docs/user-guides/advanced/experimental-mode) and requires you to [opt in](../docs/user-guides/advanced/experimental-mode#enabling-the-experimental-mode) if you want to try it out.
```

Comment:
I believe it's fine to keep this as it is (by removing link to the experimental mode doc) since at the time of v3.4.0 Podman was still in the experimental mode.

Reply:
I'm reverting it, and only removing links